### PR TITLE
fix: broken legacy event cleanup

### PR DESCRIPTION
**Pull request by FriedrichWilken**
**Description**

This PR fixes the problem with non-alphanumeric characters in legacy event requests which where supposed to be "cleaned up" automatically.

**Related issue**
see #12411

### CODE REVIEW

**Comment by FriedrichWilken** [comment]
/halt

needs more manual testing

**Comment by FriedrichWilken** [comment]
seems to be working
```
> POST /varkes-test/v1/events HTTP/1.1
> Host: localhost:8081
> User-Agent: curl/7.77.0
> Accept: */*
> Content-Type: application/cloudevents+json
> Content-Length: 192
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Content-Type: application/json
< Date: Mon, 10 Jan 2022 15:18:33 GMT
< Content-Length: 76
<
{"event-id":"358cb04e-f5e2-4cbb-b872-3ad98b2cdcdb","status":"","reason":""}
* Connection #0 to host localhost left intact
```

File: components/event-publisher-proxy/pkg/legacy-events/legacy.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/tracing"
+	//_type "google.golang.org/genproto/googleapis/actions/sdk/v2/interactionmodel/type"
```
Comment (mfaizanse) [inline components/event-publisher-proxy/pkg/legacy-events/legacy.go:12]: please remove this comment.

**Pull request merged**